microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,300 @@
+
+1. To complete your attestation and allow continued access to the Ad Selection API, create a JSON file named `attestations.json` for the domain and make it available at the `/.well-known/attestations.json` path for that domain.  For example: `https://contoso.example/.well-known/ad-selection-attestations.json`.
+
+   The `attestations.json` JSON file must be published within **30 days** of receiving the OT token.
+
+
+<!-- ------------------------------ -->
+#### Example attestations.json file
+
+The `attestations.json` file for a domain must conform to the following standards below.  The file must include the generated OT token for the top-level domain, as well as a list of APIs (features that are associated with the Ad Services API).
+
+The following is an example `attestations.json` file:
+
+```json
+{
+    "ad_selection_api_attestations": [
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"ownership_token":` is the OT token that was generated when registering this domain for the **Ad Selection API** origin trial.
+
+* In the `"platform_attestations":` section, `"platform":` must be `"edge"` or `"android"`.
+
+* In the `"platform_attestations":` section, valid members of `"attestations":` are the following:
+   * `"attribution_reporting_api":` - Part of the Ad Selection API, for attribution reporting.
+   * `"shared_storage_api":` - Part of the Ad Selection API, for shared storage.
+   * `"private_aggregation_api":` - Part of the Ad Selection API, for private aggregation.
+   * `"ad_selection_api":`
+   <!-- todo: is this list complete?  eg is there "fenced_frames_api": {} ?  per list in flag desc -->
```

Comment:
No Fenced Frame is not missing, because it's not an API, it's other part of Browser that will be used by this framework.
so you don't need to turn on / turn off it and don't need to set up it for Starting using API

Reply:
result:
![image](https://github.com/user-attachments/assets/d395ff52-7c1e-4bdf-bf79-eda5c42cc53f)